doc/tutorials/acrn_on_qemu.rst

```diff
@@ -0,0 +1,268 @@
+
+         echo 'Loading ACRN hypervisor with SDC scenario ...'
+         multiboot --quirk-modules-after-kernel /boot/acrn.32.out
+         module /boot/bzImage_sos Linux_bzImage
+      }
+
+6. Update grub ``sudo update-grub``.
+
+7. Shutdown the guest and relaunch using, virsh start ACRNSOS --console and select ACRN hypervisor from grub menu to launch SOS running on top of ACRN.
+This can be verified from dmesg as shown below,
+
+   .. code-block:: none
+
+      guestl1@ACRNSOS:~$ dmesg | grep ACRN
+      [    0.000000] Hypervisor detected: ACRN
+      [    2.337176] ACRNTrace: Initialized acrn trace module with 4 cpu
+      [    2.368358] ACRN HVLog: Initialized hvlog module with 4 cpu
+      [    2.727905] systemd[1]: Set hostname to <ACRNSOS>.
+
+8. When shutting down, make sure to cleanly destroy the SOS VM to prevent crashes in subsequent boots. This can be done using,
+
+      .. code-block:: none
+
+         virsh destory ACRNSOS, where ACRNSOS is the virsh domain name.
+
+
+SOS Networking updates for UOS
+******************************
+If you are planning to enable networking for UOS(L2 guest), the following steps might be helpful.
+
+1. Edit your :file:`/etc/netplan/01-netcfg.yaml` file to add acrn-br0 as below,
+
+   .. code-block:: none
+
+      network:
+         version: 2
+         renderer: networkd
+         ethernets:
+            enp1s0:
+               dhcp4: no
+         bridges:
+            acrn-br0:
+               interfaces: [enp1s0]
+               dhcp4: true
+               dhcp6: no
+
+2. Apply the new network configuration by,
+
+      a. cd /etc/netplan
+      #. sudo netplan generate
+      #. sudo netplan apply
+
+3. Create a tap interface (tap0) and add the tap interface as part of the acrn-br0.
+Copy the following files to :file:`/usr/lib/system/network`,
+
+- `50-acrn.network <https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/misc/acrnbridge/acrn.network>`__
+- `50-tap0.netdev <https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/misc/acrnbridge/tap0.netdev>`__
+
+4. Restart ACRNSOS guest (L1 guest) to complete the setup and start with bring-up of UOS
+
+
+Bring-up UOS (L2 Guest)
+***********************
+1. Build the device-model, using make devicemodel and copy acrn-dm to ACRNSOS guest (L1 guest) directory ``/usr/bin/acrn-dm``
+
+   .. note::
+      It should be already built as part of .._Install ACRN Hypervisor.
+
+2. On the ACRNSOS guest, install shared libraries for acrn-dm (if not already installed).
+
+   .. code-block:: none
+
+      sudo apt-get install libpciaccess-dev
+
+3. Install latest `IASL tool <https://acpica.org/downloads>`_ and copy the binary to ``/usr/sbin/iasl``.
+For this setup, used IASL 20200326 version but anything after 20190215 should be good.
+
+4. Build ACRN UOS Kernel by,
+
+      a. Clone the repo, https://github.com/projectacrn/acrn-kernel.git
+      b. cd acrn-kernel
+      c. Copy kernel_config_uos to .config
+      d. make
```

Comment:
No need to build and install the kernel modules?

Reply:
Since the needed drivers are part of the kernel, I didn't need to install the modules

Reply:
Same here, can we add a note explaining that?